src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -920,8 +920,15 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
             'Please note that the default public IP used for VM creation will be changed from Basic to Standard '
             'in the future.')
 
-    subscription_id = get_subscription_id(cmd.cli_ctx)
+    # Breaking Change Warning, change image alias
+    if image:
+        if image == "Win2008R2SP1":
+            logger.warning(
+                '[Breaking Change] On April 04, 2023, the image deployed using "Win2008R2SP1" will reach its end of life.')
```

Comment:
```suggestion
        if image == "UbuntuLTS":
            logger.warning(
                'Since on April 04, 2023, the image deployed using "UbuntuLTS" will reach its end of life, we will remove the image alias "UbuntuLTS" in the next Build Event release. So please use the new image alias "Ubuntu2204" instead')
```

Reply:
If image `Win2008R2SP1` was already invalid, we no longer need to notify customers in the warning log because it had previously stopped working. This requires @fitzgeraldsteele help to confirm this question https://github.com/Azure/azure-rest-api-specs/pull/23261/files#r1149024082